src/briefcase-using.rst

```diff
@@ -0,0 +1,231 @@
+     
+     #. In :guilabel:`Settings`, temporarily change the location of :file:`ODK Briefcase Storage`.
+     #. Manually copy the form directory from your original storage location to the temporary location.
+     #. Replace the local form definition file with a copy of the version from your Aggregate server.
+     #. Push your form instances. 
+     #. Update :guilabel:`Settings` back to the original storage location.
+     
+.. tip::
+       
+  .. Move this to Aggregate docs, or the Agg-v-Briefcase page.
+       
+  .. _aggregate-upgrade-with-briefcase:
+       
+  By pulling data into the local Briefcase Storage location and then pushing data up to an Aggregate instance, ODK Briefcase provides a mechanism to transfer data across upgrade-incompatible versions of Aggregate.
+
+
+.. _pull-from-collect:
+
+Pulling forms from Collect
+---------------------------
+
+#. Ensure all filled-in forms are finalized. 
+
+   If you have incomplete forms that you cannot finalize before pulling into Briefcase, delete them. If you need to keep them, make a copy of :file:`/sdcard/odk` before deleting them, and restore it after you are finished.
+
+#. Create a zip archive of the entire :file:`odk` directory.
+
+   .. tip::
+   
+     You'll need to use an app for this. 
+     
+     One option is `OI File Manager <https://play.google.com/store/apps/details?id=org.openintents.filemanager>`_.
+     
+#. Connect your Android device to your computer using a USB cable and choose to mount it as a Media device.
+#. Copy the zip file you created from the Android device to your local hard drive.
+#. Once it is copied onto your local hard drive, unzip the file.
+#. In Briefcase, open the :guilabel:`Pull` tab.
+#. Select *Custom Path to ODK Directory* in the :guilabel:`pull data from` drop-down.
+#. Select the unzipped :file:`odk` folder.
+#. Click :guilabel:`Pull`.
+#. On the Android device, open ODK Collect and delete the filled-in forms.
+
+
+   .. tip::
+  
+     - You can use the *Custom path to ODK Directory* any time you want to pull forms from custom location.
+     - You can confirm that the forms have been successfully pulled into Briefcase by confirming a successful pull status or by verifying the data appearing in a :ref:`CSV export file <briefcase-export-to-csv>`.
+
+.. warning::
+
+  Briefcase cannot discriminate between duplicate form instances. After you pull completed forms into Briefcase, it is important that you delete them from Collect. Otherwise, the next time you pull in forms, you will create duplicates.
+
+          
+.. note:: 
+
+  ODK Briefcase does not support pushing blank forms to ODK Collect. Instead, :ref:`manually load the forms on your ODK Collect device <loading-forms-directly>`.
+
+     
+
+
+.. _pull-from-android2.x:
+
+.. admonition:: Pulling forms --- Android 2.x and earlier
```

Comment:
I'm not sure, but it seems like this shouldn't be in an admonition.

Reply:
I wondered about that. It wasn't originally. 

Here was my thinking:

 - It's essentially a sidebar, as Android 2.x is extremely old. 
 - If the older version's procedure required only a minor change, that info would be placed in a note (which is a subtype to admonition). Similarly, I thought of this section as like a "By the Way" addendum to the larger section. 
 - If someone were reading the document as a whole (instead of just zooming in on one topic), I'd want to provide an easy wayfinding solution to skip the now-irrelevant stuff. In other words, I kinda wanted to take the section out of the regular flow of the document.

Another option is a [sidebar](http://docutils.sourceforge.net/docs/ref/rst/directives.html#sidebar). Or we could just keep it in the regular flow of the document.

Reply:
I say we drop it entirely since it's so old. @lognaturel agrees.